src/ipc/get-usb-drives.ts

```diff
@@ -19,9 +19,20 @@ interface RawDataReturn {
   blockdevices: BlockDevice[]
 }
 
+interface MountedDevice {
+  target: string
+  source: string
+}
+
+interface FindMntRawDataReturn {
+  filesystems: MountedDevice[]
+}
+
 const DEVICE_PATH_PREFIX = '/dev/disk/by-id/'
 const USB_REGEXP = /^usb(.+)part(.*)$/
 
+const MOUNTED_USB_REGEXP = /(.*)\/usb-drive(.*)$/
```

Comment:
Do we want to hardcode this? It's possible to mount drives that won't match this.

Reply:
I based this off of how we mount things: https://github.com/votingworks/kiosk-browser/blob/main/src/ipc/mount-usb-drive.ts#L18 but I guess you can pass in a label... open to removing this check but I worry that we might unmount random things we don't want to although perhaps filtering only for vfat prevents that